src/components/WorkPanelMixin.js

```diff
@@ -23,6 +25,8 @@ export default (namespace, singular, plural, loadInitially = true) => {
 			this.stopSyncTimer();
 		},
 		computed: {
+			...Utils.mapState(['federation']),
+			...Utils.mapState(namespace, ['missing']),
 			...Utils.mapState(namespace, {data: namespace}),
 			...Utils.mapState(namespace, ['pages', 'hasMore']),
```

Comment:
```suggestion
			...Utils.mapState(namespace, {data: namespace}),
			...Utils.mapState(namespace, ['missing', 'pages', 'hasMore']),
```

Reply:
I kinda liked keeping all federation stuff together (line 28 and 29) and separating it from the rest (first two lines federation, then the rest)

Reply:
I'd like to keep it as concise as possible to avoid potential confusion, I also had to look for a second to realize it's the same namespace. We also kept it together in all other instances.

Reply:
Addressed in 84f3e767873b56696b2c6bdbc029f389632b6dd5